ControlRoomApplication/ControlRoomApplication/GUI/RTControlForm.cs

```diff
@@ -815,7 +815,6 @@ private void label1_Click(object sender, EventArgs e)
         }
 
         private void plusElaButton_Up( object sender , MouseEventArgs e ) {
-            logger.Info( "Jog PosButton MouseUp" );
```

Comment:
It looks like this might have gotten accidentally removed instead of timestamped!